Nabvar movile menu

Sirvelasque opened the pull request:
-adjust bars icon position to fixed
-create a menu directly on javascript
-added 3 items with a link that redirect to an specific part of the page
-make the menu displayable by the bars icon btn and the 3 options on the menu

Sirvelasque committed:
add javascript linters

n/a

luis-pomare committed:
Add functionality.js and create menu elements

Sirvelasque committed:
take logo out of navbar and navbar adjust

set the nav button for mobile version position to fixed
Also took the menu for the mobile version out of the view

Sirvelasque committed:
set menu items for the nav bar

Sirvelasque committed:
add display event to the mobile menu

the cointainer is initially at -120vh by display_none class, and it toggles at the same time as the display class on pressing the hamburguer icon

Sirvelasque committed:
add background and link menu items

Sirvelasque committed:
fix linter errors

spaces missing

Sirvelasque committed:
fix javascript linter errors

Sirvelasque committed:
eslent error fix

Sirvelasque committed:
fix eslents errors

medaminedev66 commented on index.html:
```diff
@@ -1,94 +1,132 @@
-          <div class="logo">Sir V</div>
-          <div class="nav_mobile-btn 
-          fas fa-bars"></div>
+          <div id="hamburguer" class="nav_mobile-btn fas fa-bars"></div>
```
- [ ] The hamburger menu button is visible on desktop size. Kindly make it invisible.

<img width="866" alt="image" src="https://user-images.githubusercontent.com/77190203/152056937-937bed40-f129-4ee6-af09-738b0b44f4c9.png">

medaminedev66 commented on css/style.css:
```diff
@@ -1,9 +1,31 @@
+#menuContainer {
+  z-index: 3;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: baseline;
+  height: 100vh;
+  width: 100vw;
+  background-color: #191765;
+  position: fixed;
+  padding: 70px 10px;
+  transition-duration: 0.5s;
+}
```
- [ ] Please ensure the mobile menu follows the Figma design:
- [ ] It would be better to add the close button to your menu so the user can close your Menu.

Your design: 
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/152057973-0b4efe2f-5ff2-43db-a654-eb1e66c2b687.png">
Figma design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/77190203/152058079-07d80059-f6c0-4c1e-bdb1-e148da0a42fc.png">

medaminedev66 commented on index.html:
```diff
@@ -1,94 +1,132 @@
       <nav class="nav">
+        
         <div class="nav_mobile">
-          <div class="logo">Sir V</div>
-          <div class="nav_mobile-btn 
-          fas fa-bars"></div>
+          <div id="hamburguer" class="nav_mobile-btn fas fa-bars"></div>
         </div>
       </nav>
     </header>
-
-    <section class="headline">
+    <a id="headline_seccion"></a>
+    <section class="headline" id="headline">
       <div class="presentation">
         <h1>Walter Velasquez</h1>
         <h2>Software Developer</h2>
       </div>
```
- [ ] I would strongly recommend fixing the width of the heading section to improve your design and make it more professional.
When I open your portfolio I have to scroll the page horizontally to find the hamburger button which is a bad user experience

Your design before scrolling the page horizontally:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/152059501-e6d6e68b-2c9e-40a6-ab28-4f540e3a2468.png">

Your design after scrolling the page horizontally:
<img width="209" alt="image" src="https://user-images.githubusercontent.com/77190203/152058884-517d9ccc-0a8f-46fd-89e5-14f5011719ca.png">

Whoistolu replied:
- [ ] [REQUIRED] As stated by the previous reviewer, kindly fix the width of the heading as I also had to scroll horizontally to locate the hamburger icon. Please fix this to make your project look more professional.

Here is an image of what it looks like after scrolling horizontally:
![Screenshot from 2022-02-02 16-19-27](https://user-images.githubusercontent.com/58692590/152183381-6e3fdda3-d067-444f-bc5c-6d8be3ea065f.png)

medaminedev66 reviewed (changes requested):
Hi @Sirvelasque and @luis-pomare , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/V

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
set desktop menu and fix mobile nav menu

changes positions in mobile nav menu position
set desktop nav items

Sirvelasque committed:
add X button and nav item arrows

to stick to Figma's design made the bars icon change into x icon
then add to right signals to the menu items

Sirvelasque committed:
fix linter errors

Sirvelasque committed:
fix variable that was previusly change

I changed the variable to cammel case on the last linter errors fix, but didn't make the change in all the uses

Whoistolu reviewed (changes requested):
Hi @Sirvelasque 

You have done a good job implementing some of the recommended changes, however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Sirvelasque committed:
Fix background width

it was making a gap on the right side of the page in mobile version

Sirvelasque committed:
fix linter error

elmar8287 reviewed (approved):
Hi @Sirvelasque, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

![](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
